modify.go

```diff
@@ -18,7 +18,7 @@ func Modify(c *cli.Context) error {
 	}
 
 	var err error
-	item_id, err := strconv.Atoi(c.Args().First())
+	item_id, err := todoist.SearchByIDPrefix(client.Store.Items, c.Args().First())
```

Comment:
It seems to prefer to operate data in `client` through `client` .

Reply:
Thanks for your feedback! I will address the issues at the nearest opportunity. However, I'm not entirely sure what you mean with this particular comment. Could you elaborate a bit or provide an example?

Reply:
Implement `SearchByIDPrefix` method on `client`, then change the name. (https://github.com/sachaos/todoist/pull/35#discussion_r159706248)

e.g.

```go
item_id_prefix := c.Args().First()
item_id, err := client.CompleteItemIDByPrefix(item_id_prefix)
```

Reply:
Ok, that's what I was thinking you meant but wanted to make sure. Thanks!